### PR TITLE
Simplify a lot.

### PR DESCRIPTION
**Pull request by Warriorrrr**
## Changes made:
- Plugin version bumped to 1.1.0
- Simplified the code a lot by using setKeepLevel and setKeepInventory
- Players will no longer keep items if they suffocate
- Players will keep their xp, thanks to the setKeepLevel

### CODE REVIEW

File: src/main/java/com/karlofduty/NoVoidLoss/DeathListener.java [hunk 1]
```diff
@@ -1,61 +1,21 @@
...
-    {
-        this.plugin = plugin;
-    }
-
-    @EventHandler(priority = EventPriority.HIGHEST)
-    public void onEvent(final PlayerDeathEvent event)
-    {
-        boolean restoreAllItems = false;
-        try {
-            restoreAllItems = event.getEntity().getLastDamageCause().getCause() == EntityDamageEvent.DamageCause.VOID
```
Comment (KarlOfDuty) [inline src/main/java/com/karlofduty/NoVoidLoss/DeathListener.java:28]: This rewrite loses one of the death reasons.

File: src/main/java/com/karlofduty/NoVoidLoss/DeathListener.java [hunk 2]
```diff
@@ -1,61 +1,21 @@
...
-                           || event.getEntity().getLastDamageCause().getCause() == EntityDamageEvent.DamageCause.SUFFOCATION;
-        }
-        catch (NullPointerException ignored) { }
-
-        final ItemEntry entry = new ItemEntry();
-        for (int i = 0; i < event.getEntity().getInventory().getSize(); ++i)
-        {
-            final ItemStack stack = event.getEntity().getInventory().getItem(i);
-            if (stack != null && !stack.getType().equals(Material.AIR) && (stack.getType().equals(Material.PLAYER_HEAD) || restoreAllItems))
```
Comment (KarlOfDuty) [inline src/main/java/com/karlofduty/NoVoidLoss/DeathListener.java:37]: Also the rewrite loses the player head protection

**Review by KarlOfDuty — CHANGES_REQUESTED** [review]

File: src/main/java/com/karlofduty/NoVoidLoss/DeathListener.java (hunk 2 above)
Comment (Warriorrrr) [reply src/main/java/com/karlofduty/NoVoidLoss/DeathListener.java:37]: Any item (including player heads) gets kept.

File: src/main/java/com/karlofduty/NoVoidLoss/DeathListener.java (hunk 1 above)
Comment (Warriorrrr) [reply src/main/java/com/karlofduty/NoVoidLoss/DeathListener.java:28]: If a player dies due to suffocation damage, they wouldn't be inside the void and they'd be able to run back and recover their items.

The only real server related case I can see is when the server crashes, and the player gets stuck inside a block due to a rollback. In this case the player can just use a pickaxe to dig themselves out.

Comment (KarlOfDuty) [reply src/main/java/com/karlofduty/NoVoidLoss/DeathListener.java:28]: Players should still be protected in suffocation deaths.

File: src/main/java/com/karlofduty/NoVoidLoss/DeathListener.java (hunk 2 above)
Comment (KarlOfDuty) [reply src/main/java/com/karlofduty/NoVoidLoss/DeathListener.java:37]: Players should only keep other items when they die from the above two reasons.

**Pull request merged**